src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -46,17 +46,15 @@ class PythonAwsLambdaSources(PythonSources):
 
 
 class PythonAwsLambdaHandlerField(StringField, AsyncFieldMixin, SecondaryOwnerMixin):
-    """Entry point to the AWS Lambda handler.
-
-    You can specify a full module like 'path.to.module:handler_func' or use a shorthand to specify a
-    file name, using the same syntax as the `sources` field, e.g. 'lambda.py:handler_func'.
-
-    You must use the file name shorthand for file arguments to work with this target.
-    """
-
     alias = "handler"
     required = True
     value: str
+    description = (
```

Comment:
Open question: should this be `description` or `help`? `help` aligns with `register('--opt')`.

I also plan to do make this same change for subsystem descriptions, and the same question will apply.

Reply:
`help` is good, I think, for consistency. And it emphasizes that this is for the end user.